pyproject.toml

```diff
@@ -78,8 +78,31 @@ lint = [
     "types-requests==2.31.0.10",
 ]
 test = [
-    "flask==3.0.0",
-    "werkzeug==3.0.1",
+    "Flask>=3.0.2",
+    "pytest>=8.0.2",
+    "pytest-cov>=4.1.0",
+    "pytest-mock>=3.12.0",
+    "Werkzeug>=3.0.1",
+]
+
+[tool.pdm.scripts]
+lint = {cmd = "ruff check --force-exclude"}
+format = {cmd = "ruff format --force-exclude"}
```

Comment:
Get rid of `--force-exclude`, which is generally only useful for pre-commit hooks.

Reply:
Done.